OpenRPA.NativeMessagingHost/myAddon2/background.js

```diff
@@ -974,3 +974,53 @@ async function downloadsOnChanged(delta) {
     if (port != null) port.postMessage(JSON.parse(JSON.stringify(message)));
 
 }
+
+async function getFocusedTab() {
+    let [tab] = await tabsquery({ active: true, lastFocusedWindow: true });
+    return tab;
+}
+
+function getCurrentBrowser() {
+    if (isChrome) return "chrome";
+    if (isFirefox) return "ff";
```

Comment:
can we use `firefox`?

Reply:
I don`t get sure. I only did keep the same behavior to get browser's name from previous functions like `downloadsOnChanged(delta)`